cmake/app/boilerplate.cmake

```diff
@@ -55,6 +55,7 @@ if(NOT BOARD)
   message(FATAL_ERROR "BOARD not set")
 endif()
 
+unset(BOARD_DIR CACHE)
```

Comment:
does not seem to do the trick, I export the BOARD to something else and rebuild, it has no effect.

Reply:
Hmm, thought I had tested that sequence ... let me do some more testing ...

Reply:
Yeah, you are right, will fix ...

Reply:
Hmm, I'm going to try a different approach.